### PR TITLE
Strict Mode: format (3)

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: Wallix/wallix-bastion/ingest/parser.yml
Comment (squioc) [inline Wallix/wallix-bastion/ingest/parser.yml:1]: I detected some failures in the following test:
- Wallix/wallix-bastion/tests/rexec.json

File: Windows/windows/ingest/parser.yml
Comment (squioc) [inline Windows/windows/ingest/parser.yml:1]: I detected failures in the following tests:
- Windows/windows/tests/Event_4932.json
- Windows/windows/tests/Event_4933.json
- Windows/windows/tests/notification_subscription.json
- Windows/windows/tests/process_8002.json
- Windows/windows/tests/process_8005.json

**Review by squioc — CHANGES_REQUESTED** [review]
Thank you for the work.
I detected issues in some tests.

File: Windows/windows/ingest/parser.yml
Comment (lvoloshyn-sekoia) [reply Windows/windows/ingest/parser.yml:1]: My bad

File: Wallix/wallix-bastion/ingest/parser.yml
Comment (lvoloshyn-sekoia) [reply Wallix/wallix-bastion/ingest/parser.yml:1]: Somehow missed this. But we don't parse anything in this test. Perhaps we should either delete it or just don't raise errors for the `parsed_event` stage?

Comment (squioc) [reply Wallix/wallix-bastion/ingest/parser.yml:1]: OK. Indeed, nothing is extracted in this test. I agree to delete it.

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**